DisguiseUnityRenderStream/Editor/Disguise.RenderStream.Editor.asmdef

```diff
@@ -2,6 +2,7 @@
     "name": "Disguise.RenderStream.Editor",
     "references": [
         "Disguise.RenderStream",
+        "Disguise.RenderStream.PipelineAbstraction",
```

Comment:
This is only for the `AddAlwaysIncludedShader` calls in `DisguiseRenderStreamBuildPreProcessor.OnPreprocessBuild`.